src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public final String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+
+        return color.toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constant, use the `name()` method. This is a more reliable way to get the string representation of an enum constant, as `toString()` can be overridden and may not return the expected result.